_posts/2019-05-31-JSoC-Cohort.md

```diff
@@ -0,0 +1,50 @@
+---
+title: JSoC Cohort
+author: Tejan Karmali, Shreyas Kowshik, Kartikey Gupta, Manjunath Bhat, Avik Pal, Raghvendra Gupta
+layout: blog
+---
+
+JSoC stands for Julia's Season of Contributions. It consists of [GSoC](https://summerofcode.withgoogle.com/organizations/5131561161195520/) students of Julia + JSoC students. (JSoC = GSoC(Julia) +JSoC :P). Flux is hosting 6 JSoC students this time. They will be working on a variety of sub-domains of ML, ranging from Differentiable Programming and RL to GAN and ray tracing; to demonstrate a fresh approach to build these models. 
+Here, we introduce our students and their projects over the summer. It’s going to be an exciting summer!
```

Comment:
Introduction could do with padding out a bit, right now it seems like it's written as notes (e.g. "Julia + JSoC students"). Start with the excitement and then go into detail.

Worth mentioning that these are a mix of students funded by Google Summer of Code and JSoC, which is run by the community with NumFOCUS funds (have explicit links).